system/jlib/jstats.cpp

```diff
@@ -1865,7 +1858,7 @@ void verifyStatisticFunctions()
     //Check the various functions return values for all possible values.
     for (unsigned i1=SMeasureAll; i1 < SMeasureMax; i1++)
     {
-        const char * prefix = queryMeasurePrefix((StatisticMeasure)i1);
+        const char * prefix __attribute__((unused)) = queryMeasurePrefix((StatisticMeasure)i1);
```

Comment:
same as queryTreeTag(kind) above, should this be deleted or a dbgassertex() added?